### PR TITLE
fix cannot replace endpoint for adulting session

### PR DESCRIPTION
**Pull request by xquanluu**
tested on jambonz.me

### CODE REVIEW

File: lib/session/adulting-call-session.js [hunk 1]
```diff
@@ -45,8 +45,10 @@ class AdultingCallSession extends CallSession {
     return this.sd.ep;
   }
 
-  /* see note above */
-  set ep(newEp) {}
+  // When aldusting session kicked from conference, replaceEndpoint is a must
```
Comment (davehorton) [inline lib/session/adulting-call-session.js:48]: typo

Comment (xquanluu) [reply lib/session/adulting-call-session.js:48]: fixed

**Comment by vdharashive** [comment]
@xquanluu can u rebase

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
please rebase

**Review by davehorton — APPROVED** [review]

**Pull request merged**